grafana/RELEASES.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## 0.7 (Unreleased)
```

Comment:
Was this meant to be here? Maybe do in another PR or commit? We should probably mention some of the GCM changes as well.